pkg/kv/kvserver/load/replica_load.go

```diff
@@ -199,6 +211,23 @@ func (rl *ReplicaLoad) Stats() ReplicaLoadStats {
 	}
 }
 
+// StatsShort returns a current stat summary of replica load using only one bucket.
+func (rl *ReplicaLoad) StatsShort() ReplicaLoadStats {
```

Comment:
This function returns the replica statistics using only one bucket of the ReplicaStats object.